canary/consumer-java/src/main/java/com/amazon/kinesis/video/canary/consumer/WebrtcStorageCanaryConsumer.java

```diff
@@ -0,0 +1,147 @@
+package com.amazon.kinesis.video.canary.consumer;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.lang.Exception;
+import java.util.Date;
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.ArrayList;
+
+import com.amazonaws.auth.SystemPropertiesCredentialsProvider;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchAsync;
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchAsyncClientBuilder;
+import com.amazonaws.services.kinesisvideo.AmazonKinesisVideo;
+import com.amazonaws.services.kinesisvideo.AmazonKinesisVideoClientBuilder;
+import com.amazonaws.services.kinesisvideo.model.APIName;
+import com.amazonaws.services.kinesisvideo.model.GetDataEndpointRequest;
+import com.amazonaws.services.kinesisvideo.model.TimestampRange;
+import com.amazonaws.services.kinesisvideo.model.FragmentSelector;
+import com.amazonaws.services.cloudwatch.model.Dimension;
+import com.amazonaws.services.cloudwatch.model.MetricDatum;
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
+import com.amazonaws.services.cloudwatch.model.StandardUnit;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class WebrtcStorageCanaryConsumer {
+
+    private static void calculateFragmentContinuityMetric(CanaryFragmentList fragmentList, Date canaryStartTime, String streamName, String canaryLabel, SystemPropertiesCredentialsProvider credentialsProvider, String dataEndpoint, String region){
+        try{
+            TimestampRange timestampRange = new TimestampRange();
+            timestampRange.setStartTimestamp(canaryStartTime);
+            timestampRange.setEndTimestamp(new Date());
+
+            FragmentSelector fragmentSelector = new FragmentSelector();
+                fragmentSelector.setFragmentSelectorType("SERVER_TIMESTAMP");
+                fragmentSelector.setTimestampRange(timestampRange);
+
+            Boolean newFragmentReceived = false;
+
+            ExecutorService executorService = Executors.newFixedThreadPool(10);
+            Future<List<CanaryFragment>> listFragmentResult = executorService.submit(new CanaryListFragmentWorker(streamName, credentialsProvider, dataEndpoint, Regions.fromName(region), fragmentSelector));
+            List<CanaryFragment> newFragmentList = listFragmentResult.get();
+
+            if (newFragmentList.size() > fragmentList.getFragmentList().size())
```

Comment:
Lets have the formatting consistent across the Java files. If you open the code in Intellij, you can do this easily. Follow the instructions here: https://www.jetbrains.com/idea/guide/tips/reformat-file/#:~:text=You%20can%20use%20%E2%8C%98%E2%8C%A5,L%20(Windows%2FLinux).